### PR TITLE
Update readme, Code QA, import structure

### LINKED ISSUE #2: Readme and Project structure updates
1. Credit section: do mention raw2temp in ThermImage-R (https://github.com/gtatters/Thermimage), exiftool, flyr
2. When user clicks on the badge, can it redirect to the output on the Github workflows page ?(i.e. a commit-independent version of https://github.com/detecttechnologies/thermal_base/runs/3123657725?check_suite_focus=true)
3. In the `Usage` section, it's `from thermal_base.thermal_base import ThermalImage`. Is there any way we can make it `from thermal_base import ThermalImage`? Perhaps defining something in the `__init__.py` file?
4. In the `Usage` section, there's also a `from thermal_base import utils`. Can a user use the library without importing that library?
5. The `Disclaimer` section of the readme can be changed to `Supported formats`. Additionally, its content can be changed to something along the lines of

|  Data Format  |  Sample Cameras  |  Support  |
| --- | --- | --- |
|  FLIR RJPG with TIFF-format of thermal embedding |  Zenmuse XT-2 | ✅ |
|  FLIR RJPG with TIFF-format of thermal embedding |  FLIR E-4, T660 | ✅ |
|  DJI-encoded thermal image | Zenmuse H20-T | ✅ |
|  FLIR SEQ Thermal Video |  Zenmuse XT-2 | ✅ |
|  FLIR CSQ Thermal Video |  Zenmuse XT-2 | ❌ |

*RJPG is also known as R-JPEG

### PR DESCRIPTION
**Pull request by mukheshpugal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 A python package for decoding and common processing for thermographs / thermograms. Currently supports FLIR's TIFF and PNG encoded metadata, and DJI-encoded metadata
 
-![Quality check](https://github.com/mukheshpugal/thermal_base/actions/workflows/qualitycheck.yml/badge.svg)
+[![Quality check](https://github.com/detecttechnologies/thermal_base/actions/workflows/qualitycheck.yml/badge.svg)](https://github.com/detecttechnologies/thermal_base/actions)
 
 ## Install
 1. This tool requires exiftools to be installed.
```
Comment (rsnk96) [inline README.md:8]: Can you also add Windows download link?

File: README.md [hunk 2]
```diff
@@ -19,13 +19,20 @@ from thermal_base import utils
 image = ThermalImage(image_path="path/to/image", camera_manufacturer="dji/flir")
```
Comment (rsnk96) [inline README.md]: Can you split it into two examples? 
1. One which just shows the thermal image decoding, and then maybe a couple of key variables (ex: the temperature matrix, and the sensor values)
2. One which shows utils from ThermalImageHelpers ( `from thermal_base import utils`) being used

File: README.md [hunk 3]
```diff
@@ -19,13 +19,20 @@ from thermal_base import utils
...
 ```
```
Comment (rsnk96) [inline README.md:30]: Can you also change this to `from thermal_base import ThermalImage` now that the __init__ file has been modified??

File: README.md [hunk 4]
```diff
@@ -19,13 +19,20 @@ from thermal_base import utils
...
 
-**Disclaimer**
-
-Tested against data from: 
-* Flir encoding format: DJI XT-2, Flir E-40 (Temperature values go out of the image frame if the image height is <255 pixels)
-* DJI-encoding format: DJI Zenmuse H20-T
-* Thermapp (available in past commits)
+## Supported formats
```
Comment (rsnk96) [inline README.md:34]: In the `Usage` section, can you also add a mention that this repo can be used in conjunction with Thermal-Image-Analysis repo for an interactive experience

Comment (rsnk96) [reply README.md:34]: And similarly, in the other repo, can you mention that it relies on this repo for a backend?

File: .github/workflows/qualitycheck.yml [hunk 5]
```diff
@@ -27,11 +27,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install black pylama
-    - name: Black
-      run: |
-        black --diff --color --line-length 119 thermal_base
-        black --check --line-length 119 thermal_base
+        python -m pip install pylama
```
Comment (rsnk96) [inline .github/workflows/qualitycheck.yml:30]: Was black causing issues?

Comment (mukheshpugal) [reply .github/workflows/qualitycheck.yml:30]: Yeah. Black and pylama sorted imports differently

**Pull request merged**